homework/PavelBo/exceptions/knowingExceptions/src/main/java/com/tutrit/java/quickstart/service/Verification.java

```diff
@@ -0,0 +1,70 @@
+package com.tutrit.java.quickstart.service;
+
+import com.tutrit.java.quickstart.exceptions.ArrayLastValueZeroRuntimeException;
+import com.tutrit.java.quickstart.exceptions.ArrayOddValueException;
+import com.tutrit.java.quickstart.exceptions.ArrayLastValueZeroException;
+import com.tutrit.java.quickstart.exceptions.ArrayOddValueRuntimeException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class Verification {
+    Logger log = LoggerFactory.getLogger(Verification.class);
+
+    public void arrayIndexOutOfBound(int i){
```

Comment:
I don't like name these method. Maybe it seems to me. I don't sure, but I can 
call your attention on it.